examples/basic_usage.py

```diff
@@ -295,6 +295,20 @@ def my_func(x):
 # %% [markdown]
 # _Stay tuned for some new features!_
 
+# %% [markdown]
+# ---
+# # Cross-validation with skore
```

Comment:
See my notes above. (I guess I should have commented this file instead!)

Reply:
done